### DISCUSSION TITLE
Sending video clips instead of images

### DISCUSSION THREAD

**Discussion by louispeterson22** [AxisCommunications/acap-integration-examples-azure#18] [Q&A]
The instructions for all three examples are wonderful, thank you for these guides!  

In the guide for sending images to Azure storage, does anyone know if sending video clips should work as well?  I have tried using the 'send video clip through HTTPS' option for the event instead of the images one without success, but that could just be me doing something wrong.

**Comment by mattias-kindborg-at-work** [comment]
Hi @louispeterson22 and welcome! Let me first start by apologizing for the incredible late answer. This response time is not acceptable, and we will start to behave like mindful and responsible open source citizens.

Did you succeed with sending video clips to a storage account, or is this something you still are struggling with?

**Reply by louispeterson22** [reply]
Hi @mattias-kindborg-at-work ! No worries on the response, I appreciate all the documentation that you and Axis provide on your Github!

I have not been successful in sending video clips to Azure Blob storage.  I had messed around with it back when I had initially posted, but figured that object storage might not work for actively writing a video file to.  As opposed to sending a still frame that would be a one-time write.

It is more of a curiosity for me, to see if it is possible.  I'm more invested in the ACAP development and doing analytics on the camera than sending clips to the cloud.  If I find that I do have to send clips to the cloud for some analytic, I will probably just use an instance as a file server, run my analytic there, and store the end result in Blob storage or a database service.
